com.unity.cinemachine/Samples~/2D Samples/Prefabs/Camera.prefab

```diff
@@ -0,0 +1,773 @@
+%YAML 1.1
```

Comment:
I'm not a fan of this prefab.  It's doing too much and will be confusing to the novice.  
It's combining the light, the camera, and the canvas.  There should be a separation of concerns.
The light should be part of the stage set, not the camera.
The camera should be its own thing, with the brain, clearly visible in the hierarchy at the top level.
You can put the perspective switcher canvas thing in its own prefab, which we can choose to add per-scene if we want